src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs

```diff
@@ -26,14 +21,13 @@ public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
             _originPhaseName = GetPhaseName();
             phaseName = _originPhaseName == null ? phaseName : _originPhaseName + "." + phaseName;
             SetPhaseName(phaseName);
-            if (enablePerformanceScope)
-            {
-                _performanceScope = new PerformanceScope("Scope:" + phaseName, LogLevel.Diagnostic);
-            }
         }
 
+        public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
```

Comment:
maybe the same question: It seems like the boolean parameter do not play any roles, why don't we remove that?

---
In reply to: [95773945](https://github.com/dotnet/docfx/pull/1168#discussion_r95773945) [](ancestors = 95773945)